client/messaging/common-message-layout.test.tsx

```diff
@@ -0,0 +1,328 @@
+import React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+import { SbUserId } from '../../common/users/user-info'
+import { TextMessage } from './common-message-layout'
+
+describe('client/messaging/common-message-layout/TextMessage', () => {
+  const renderer = ShallowRenderer.createRenderer()
+  const doMatch = (text: string): React.ReactElement => {
```

Comment:
nit: this isn't really doing any matching, maybe `doRender` or something would be more appropriate